controllers/release/controller.go

```diff
@@ -105,12 +101,6 @@ func (c *Controller) Register(mgr ctrl.Manager, log *logr.Logger, _ cluster.Clus
 				Kind:  "Release",
 				Group: "appstudio.redhat.com",
 			},
-		}, builder.WithPredicates(predicates.GenerationUnchangedOnUpdatePredicate{}, gitops.DeploymentFinishedPredicate())).
-		Watches(&tektonv1.PipelineRun{}, &libhandler.EnqueueRequestForAnnotation{
```

Comment:
It can be changed to -
For(&v1alpha1.Release{}, builder.WithPredicates(predicate.GenerationChangedPredicate{}, predicates.IgnoreBackups{})).
		Watches(&tektonv1.PipelineRun{}, &libhandler.EnqueueRequestForAnnotation{
			Type: schema.GroupKind{
				Kind:  "Release",
				Group: "appstudio.redhat.com",
			},
                      ...

Reply:
Sloppy change. The wrong `Watches` was removed.

Reply:
Thanks for a great catch @jinqi7 , fixed now!